rocketpy/Rocket.py

```diff
@@ -278,8 +309,138 @@ def __init__(
         # Evaluate static margin (even though no aerodynamic surfaces are present yet)
         self.evaluateStaticMargin()
 
+        self.evaluateDragCoefficient()
+
         return None
 
+    def evaluateDragCoefficient(self):
+        """Calculates and returns the drag coefficient as the sum of
+        of the individual drag coefficients at zero angle of attack
+        as a function of the Mach number. The angle of attack is
+        assumed to be zero.
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        -------
+        self.viscous_friction_coefficient : Function
+            Function of mach number expressing the viscous friction coefficient,
+            defined as in the literature.
+        """
+        self.evaluateViscousFrictionCoefficient()
+        self.evaluateForebodyDragCoefficient()
+
+        self.drag_coefficient_estimate = self.forebody_drag_coefficient + np.sum(
```

Comment:
One problem that I saw is that when I run "Rocket.allInfo()", the graphs of Cd x Mach has a title of "Scalar x Scalar". I believe some changes on Function objects could adjust that behavior.